### PR TITLE
Update Get-CsOnlineDirectoryTenant.md - Deprecation Info

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Get-CsOnlineDirectoryTenant [[-Tenant] <Guid>] [-DomainController <Fqdn>] [-Forc
 ```
 
 ## DESCRIPTION
+*Note: Starting with TeamsPowerShell Module 4.0 this cmdlet will be deprecated. Users are required to use the Get-CsTenant cmdlets to view information previously present in Get-CsOnlineDirectoryTenant*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md:25]:
```suggestion
**Note**: Starting with Teams PowerShell Module 4.0, this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Comment by isrumnon** [comment]
> Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

Done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — COMMENTED** [review]
Hi @isrumnon,

When 4.0 is arriving? Should we add this here in advance? Thanks!

**Comment by isrumnon** [comment]
> Hi @isrumnon,
> 
> When 4.0 is arriving? Should we add this here in advance? Thanks!

@dariomws done!

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ Get-CsOnlineDirectoryTenant [[-Tenant] <Guid>] [-DomainController <Fqdn>] [-Forc
 ```
 
 ## DESCRIPTION
+**Note**: Starting with Teams PowerShell Module 4.0 (available on 1st March'22), this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md:25]:
```suggestion
**Note**: Starting with Teams PowerShell Module 4.0 (available from March 1, 2022), this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@isrumnon, minor edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Pull request merged**